lib/ClosureF.hs

```diff
@@ -67,12 +67,11 @@ data Expr t e =
    | JNew ClassName [Expr t e]
    | JMethod (Either ClassName (Expr t e)) MethodName [Expr t e] ClassName
    | JField  (Either ClassName (Expr t e)) FieldName ClassName
-   | PolyList [Expr t e] (Type t)
-   | JProxyCall (Expr t e) (Type t)
    | SeqExprs [Expr t e]
    | Data S.RecFlag [DataBind t] (Expr t e)
    | Constr (Constructor t) [Expr t e]
    | Case (Expr t e) [Alt t e]
+   | Error (Type t) (Expr t e)
```

Comment:
I see. If you have an idea how to fix #108 without adding the extra case in ClosureF, it'd be good. There's going to be the compiler meeting tomorrow (or today already), so we can discuss this issue there.